mlir-sycl/lib/Dialect/IR/SYCLOpsDialect.cpp

```diff
@@ -93,8 +93,6 @@ class SYCLOpAsmInterface final : public mlir::OpAsmDialectInterface {
       return "drw";
     case mlir::sycl::MemoryAccessMode::Atomic:
       return "ato";
-    default:
```

Comment:
To fix compile warnings

Reply:
But then if one enumerator is added the unreachable is not triggered? Might be able to suppress the compiler warning that shows up if all enumerators are covered (not sure exactly how right now).

Reply:
This change is unrelated to this PR, undo.